src/lib/iterator.js

```diff
@@ -2,15 +2,16 @@
 
 const { readHeader, readBlockHead, asyncIterableReader } = require('@ipld/car/decoder')
 
-const { RAW_BLOCK_CODEC, decodeBlocks } = require('./config')
+const { RAW_BLOCK_CODEC } = require('../config')
 
 class CarIterator {
-  constructor(version, roots, reader, length) {
+  constructor(version, roots, reader, length, decodeBlocks) {
```

Comment:
Please call this something else like `readBlockData` or something. This class does not do any decoding.

Reply:
I'd leave as it is, to be sure what it does and come from